apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx

```diff
@@ -236,13 +236,19 @@ const EditMusicLevelData: React.FunctionComponent<EditMusicLevelDataProps> = ({
         </div>
       </CollapsibleSection>
       <hr />
+      <div>
+        {'You can also edit toolbox blocks using Blockly using Extra Links.'}
+      </div>
       <CollapsibleSection headerContent="Toolbox">
         <EditMusicToolbox
           toolbox={levelData.toolbox}
           blockMode={levelData.blockMode || BlockMode.SIMPLE2}
           addFunctionDefinition={levelData.toolbox?.addFunctionDefinition}
           addFunctionCalls={levelData.toolbox?.addFunctionCalls}
-          onChange={toolbox => setLevelData({...levelData, toolbox})}
+          onChange={toolbox =>
+            // Reset toolbox mode configuration when changing toolbox settings
+            setLevelData({...levelData, toolbox, toolboxDefinition: undefined})
```

Comment:
This is important because the `toolboxDefinition` would be prioritized over the `toolbox` if both continued to exist, meaning users potentially wouldn't see changes here reflected in the level.